pkgs/development/python-modules/deepdiff/default.nix

```
@@ -2,44 +2,60 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
+  pythonOlder,
+
+  # build-system
+  setuptools,
+
+  # dependencies
   click,
-  ordered-set,
+  orderly-set,
   orjson,
+
+  # optional-dependencies
   clevercsv,
+
+  # tests
   jsonpickle,
   numpy,
   pytestCheckHook,
   python-dateutil,
   pyyaml,
   toml,
   tomli-w,
-  pythonOlder,
+  polars,
+  pandas,
 }:
 
 buildPythonPackage rec {
   pname = "deepdiff";
-  version = "7.0.1";
-  format = "setuptools";
+  version = "8.0.0";
+  pyproject = true;
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "seperman";
     repo = "deepdiff";
     rev = "refs/tags/${version}";
```

```suggestion
    rev = "6d8a4c7c32d5ac57919955954790be994d01fe57";
```
until https://github.com/seperman/deepdiff/issues/497 is resolved

---

> until [seperman/deepdiff#497](https://github.com/seperman/deepdiff/issues/497) is resolved

Good idea, done :).